README.md

```diff
@@ -22,14 +22,14 @@ The scripts are run from the `scripts` directory.
 
 1. Clone this repo.
 
-2. Install Cloudinary gem. 
+2. Install the Cloudinary gem: 
 
 ```bash 
 `gem install cloudinary`
 ```
 3. Set up credentials.
 
-Create a `./scripts/config.rb` file and copy the value from the ClOUDINARY_URL located in the Cloudinary [Management Console](https://cloudinary.com/console) into the file as an argument for the `config_from_url` command.  See the `config.rb.sample`.  The code below shows two alternatives for adding credentials.
+Create a `./scripts/config.rb` file and copy the ClOUDINARY_URL value from your Cloudinary [Management Console](https://cloudinary.com/console) into the file as an argument for the `config_from_url` command.  See the [config.rb.sample](/scripts/config.rb.sample).  The code below shows two alternatives for adding credentials.

```

Comment:
These instructions assume you are choosing the 2nd of the two options shown below. 
Perhaps makes more sense to have these instructions (without mentioning "two alternatives"), then show just the Cloudinary.config_from_url syntax.

And then below that, say "Alternatively, you can configure each of the required parameters separately as follows: 
And then show the Cloudinary.config do option...